Find a functionally equivalent code:android.view.Window.setFormat:COMMENT
Method Modifier: public      
Comment:/**
 * Set the format of window, as per the PixelFormat types.  This overrides
 * the default format that is selected by the Window based on its
 * window decorations.
 *
 * @param format The new window format (see PixelFormat).  Use
 * PixelFormat.UNKNOWN to allow the Window to select
 * the format.
 *
 * @see PixelFormat
 */

@@ -7,13 +7,11 @@
         attrs.format = mDefaultWindowFormat;
         mHaveWindowFormat = false;
     }
-    if (mCallback != null) {
-        mCallback.onWindowAttributesChanged(attrs);
-    }
+    dispatchWindowAttributesChanged(attrs);
 }

